src/result.rs

```diff
@@ -81,6 +89,35 @@ impl QueryResultWrapper {
       .map(|f| f.name.clone())
       .collect()
   }
+
+  #[napi]
+  pub fn get_columns_specs(&self) -> Vec<MetaColumnsWrapper> {
+    self
+      .internal
+      .col_specs()
+      .iter()
+      .map(|f| {
+        MetaColumnsWrapper {
+          // ToDo: Check if rust provides such information
+          ksname: "unknown?".to_string(),
+          tablename: f.table_spec.table_name().to_string(),
+          name: f.name.clone(),
+          type_code: CqlTypes::Ascii,
+        }
+      })
+      .collect()
+  }
+
+  #[napi]
+  pub fn get_warnings(&self) -> Vec<String> {
+    self.internal.warnings.clone()
+  }
+
+  #[napi]
+  // As we don't have UUID type, we temporary use string
+  pub fn get_trace_id(&self) -> Option<String> {
+    self.internal.tracing_id.map(|f| f.to_string())
+  }
```

Comment:
I'm not happy with this...
But it's not very important. Please add `// FIXME: ` here, though.